configs/analysis/forms_monthly.yml

```diff
@@ -0,0 +1,5 @@
+forms_monthly:
+  factor_start: .5
+  factor_end: 1.5
+  factor_step: .5
+  field_name: factor
```

Comment:
Can you add some docs about how this works and what the fields are.

Reply:
https://github.com/dimagi/commcare_resource_model/pull/25/commits/18c1627ed7495617e6c8eaf791ea5e23e1acc0b5